android/java/org/chromium/chrome/browser/app/domain/PortfolioModel.java

```diff
@@ -0,0 +1,164 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Comment:
Let's switch to HTTPS
```suggestion
 * You can obtain one at https://mozilla.org/MPL/2.0/. */
```